src/components/ArticlePage.jsx

```diff
@@ -23,7 +27,21 @@ const ArticlePage = () => {
     });
   }, [article_id]);
 
-  console.log(comments[0]);
+  const handleCroak = (value) => {
```

Comment:
As much as I like the vibe of the croak - this name only makes sense if you've looked at your app. I'd probably make it more generic like handleVote. Sorry to be a party pooper